packages/calendar/src/index.tsx

```diff
@@ -0,0 +1,76 @@
+import * as React from 'react'
+import {useState} from 'react'
+import getUserLocale from 'get-user-locale'
+import {useDefaults} from '@smashing/theme'
+import {CalendarProps, CalendarAppearanceType} from './types'
+import {StyledContainer, StyledInput, StyledCalendar} from './styles'
+
+const Calendar: React.FC<CalendarProps> = ({
+  onChange = (date: Date) => {},
+  value,
+  ...props
+}) => {
+  const [open, setOpen] = useState(false)
+  const defaults = useDefaults('calendar', props, {
+    appearance: 'default' as CalendarAppearanceType
+  })
+
+  const getTwoDigitNumber = variable => {
+    return variable > 9 ? variable : '0' + variable
+  }
+
+  const getFormattedDate = () => {
+    if (!value) {
+      return ''
+    }
+    const day = value.getDate()
+    const month = value.getMonth() + 1
+    const year = value.getFullYear()
+
+    return `${getTwoDigitNumber(day)}/${getTwoDigitNumber(month)}/${year}`
```

Comment:
Simpler and locale aware solution but I don't know how to get pattern `DD/MM/YYYY` which you use later
```suggestion
    return value.toLocaleDateString()
```